src/handlers/shared/start.ts

```diff
@@ -13,7 +13,7 @@ export async function start(context: Context, issue: Context["payload"]["issue"]
   const assignee = context.payload.issue.user.login;
   const userRole = await getUserRole(context, assignee);
   const { maxConcurrentTasks } = config.miscellaneous;
-  const maxTask = maxConcurrentTasks.find(({ role }) => role.toLowerCase() === userRole)?.limit as number;
+  const maxTask = maxConcurrentTasks.find(({ role }) => role.toLowerCase() === userRole.toLowerCase())?.limit as number;
```

Comment:
this isn't needed as you are doing this inside `getUserRole()` already, you can take the `limit` from the function return data

Reply:
Removed Thanks!